packages/mask/src/plugins/Trader/SNSAdaptor/trending/TrendingView.tsx

```diff
@@ -229,10 +235,10 @@ export function TrendingView(props: TrendingViewProps) {
 
         return (
             <ActionButton sx={{ marginTop: 1 }} color="primary" onClick={() => setStorage(nextOption)}>
-                Switch to {resolveDataProviderName(nextOption)}
+                Switch to {resolveSourceTypeName(nextOption as unknown as SourceType)}
```

Comment:
I think we can still use `resolveDataProviderName`.

Reply:
Please refactor it as a i18n field.

Reply:
done